### PR TITLE
add arima 2readbamcombiner nf-core module

### PR DESCRIPTION
**Pull request by gq1**
## PR checklist

Closes #XXX 

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
    - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd`

### CODE REVIEW

File: modules/arima/2readsbamcombiner/meta.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+description: Combin two paired single read bam files into one sorted bam
+keywords:
+  - samtools
+  - combine
+  - sort
+  - arima
+tools:
+  - arima:
+      description: Combin paired subgke read bam files into one sorted bam using Samtools and Perl script
```
Comment (muffato) [inline modules/arima/2readsbamcombiner/meta.yml:11]:
```suggestion
      description: Combine paired single read bam files into one sorted bam using Samtools and Perl script
```

File: modules/arima/2readsbamcombiner/meta.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+name: arima_2readsbamcombiner
+
+description: Combin two paired single read bam files into one sorted bam
```
Comment (muffato) [inline modules/arima/2readsbamcombiner/meta.yml:3]:
```suggestion
description: Combien two paired single read bam files into one sorted bam
```

File: modules/arima/2readsbamcombiner/meta.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+keywords:
+  - samtools
+  - combine
+  - sort
+  - arima
+tools:
+  - arima:
+      description: Combin paired subgke read bam files into one sorted bam using Samtools and Perl script
+      homepage: https://arimagenomics.com/
+      documentation: https://usermanual.wiki/Document/ArimaMappingUserGuideA160156v01.424413804/view
```
Comment (muffato) [inline modules/arima/2readsbamcombiner/meta.yml:13]: Do you know if this file was uploaded by Arima themselves ? If you're not sure, better to use the version that is on their GitHub
```suggestion
      documentation: https://github.com/ArimaGenomics/mapping_pipeline/blob/master/Arima_Mapping_UserGuide_A160156_v02.pdf
```

File: modules/arima/2readsbamcombiner/meta.yml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+      tool_dev_url: https://github.com/ArimaGenomics/mapping_pipeline/blob/master/two_read_bam_combiner.pl
+
```
Comment (muffato) [inline modules/arima/2readsbamcombiner/meta.yml:15]:
```suggestion
      licence: MIT

```

**Comment by gq1** [comment]
We can wait to merge this, when I push docker image to public

File: modules/arima/2readsbamcombiner/meta.yml [hunk 5]
```diff
@@ -0,0 +1,47 @@
+name: arima_2readsbamcombiner
+
+description: Combine two paired single read bam files into one sorted bam
+keywords:
+  - samtools
```
Comment (muffato) [inline modules/arima/2readsbamcombiner/meta.yml:5]:
```suggestion
  - bam
  - samtools
```

**Comment by gq1** [comment]
@muffato every new module will modify pytest file, we will see more merge conflicts, especially pull request from our main branch to nf-core master branch.

**Pull request merged**